### ISSUE TITLE
LNK1104: cannot open file 'C:\Qt\Qt5.4.1\5.4\msvc2013_64\lib\Qt5AVWidgetsd.lib'

### ISSUE THREAD

**Issue by dakom** [wang-bin/QtAV#439]
Using QtCreator with MSVC2013

The file 'C:\Qt\Qt5.4.1\5.4\msvc2013_64\lib\Qt5AVWidgetsd.lib' exists on the drive (evidently through build and sdk_install.bat completing successfully)

If there's any more info I can provide, please let me know. Here's my source code in case it makes a difference:

```
#include <QApplication>
#include <QDesktopWidget>
#include <QStyle>

#include <QtAV/QtAV.h>
#include <QtAV>
#include <QtAVWidgets>

int main(int argc, char *argv[])
{
    QApplication app(argc, argv);
    QtAV::GLWidgetRenderer2 renderer;

    QtAV::Widgets::registerRenderers();

    renderer.show();

    QtAV::AVPlayer player;
    player.setRenderer(&renderer);
    player.play("test.avi");

    return app.exec();
}
```

**Comment by xiaoweiChen** [comment]
yeah, the same issue also happen on my machine. In installer directory, I find the static libs is all `.a` file, this file can not be supported by windows compiler. So, the author can provide the `.lib` runtime libraries for the developers who use Microsoft compiler? Thanks!

**Comment by BrosDs** [comment]
I have the same issue too, but in my case in C:\Qt\5.4\msvc2013_64\lib (my path) I can't see the lib

**Comment by wang-bin** [comment]
I did not test debug build. I will try.

**Comment by dkomer** [comment]
Thanks... how would I build in non-debug mode?

**Comment by wang-bin** [comment]
set release mode in qtcreator. qtcreator's default mode debug

**Comment by dkomer** [comment]
Thanks- will try that when I get home later :)

**Comment by dkomer** [comment]
Thank you, release build works. Happy to test debug build when you've confirmed that it works on your end

**Comment by wujialiang** [comment]
你好，我在windows上编译，编译器是mingw32，qt版本是5.4.1。编译完成qtav的库并且成功安装。
然后编译自己的程序，release可以通过，但是运行的时候报出【plugin cannot be loaded for module "QtAV": ????????C:/Qt/Qt5.4.1/5.4/mingw491_32/qml/QtAV/QmlAV.dll?????????????????】
我检查过，此库文件是存在的，但是就是读取不到，我直接安装qtav的安装包后来编译就可以通过，请问是什么问题呢？谢谢。

**Comment by wang-bin** [comment]
try the latest code. run sdk_uninstall.bat before build and sdk_install.bat after build

**Comment by nhatson1987** [comment]
I used release branch http://downloads.sourceforge.net/project/qtav/release/1.6.0/QtAV1.6.0-VS2013x64.exe

The release build works. Test app works perfectly

The debug build is missing Qt5AVWidgetsd.lib file. 
Qt5AVWidgetsd.lib doesn't exist in the installation package but Qt5AVWidgets1.lib does. 
Is this a bug or typos?

I renamed Qt5AVWidgets1.lib to Qt5AVWidgetsd.lib, the debug compilation works but when I try to run the program it said: 
"QWidget: Must construct a QApplication before a QWidget"
